### ISSUE TITLE
virtualenv-2.7 no longer exists

### ISSUE THREAD

**Issue by Bekt** [pypa/virtualenv#692]
Hi, before `pip install virtualenv` used to create `virtualenv` and `virtualenv-2.7` executables. But today, I noticed the pip command creates `virtualenv` and `virtualenv-3.4`. What is the reason for this, and how can I get virtualenv-2.7?

My environment:
```
$ ls /usr/local/bin/virtualenv*
/usr/local/bin/virtualenv     /usr/local/bin/virtualenv-3.4
$ virtualenv --version
12.0.4
$ pip --version
pip 6.0.3 from /Library/Python/2.7/site-packages (python 2.7)
$ python --version
Python 2.7.5
```

**Comment by glarrain** [comment]
The disappearance of `virtualenv-3.4` bit me. It probably should have been documented as a backwards incompatible change so people knew that they should check whether to make changes to their code (for example deployment scripts).

**Comment by dstufft** [comment]
Fixed (by removal) in #852
